### PR TITLE
feat: automate creation of the first LTS release

### PR DESCRIPTION
**Pull request by richardlau**
Add an option to `git node release` that marks the release being
created as the transition from Current to LTS.

cc @nodejs/releasers

### CODE REVIEW

**Comment by richardlau** [comment]
See https://gist.github.com/richardlau/17d87552525d4e05da1298163fdf7cb7 for a sample session (creating `v14.15.0-proposal` for https://github.com/nodejs/node/pull/35746).

File: lib/prepare_release.js [hunk 1]
```diff
@@ -249,6 +268,14 @@ class ReleasePreparation {
     ]).trim();
   }
 
+  async getLTSCodename(version) {
+    const { cli } = this;
+    return await cli.prompt(
+      'Enter the LTS code name for this release line:',
+      { questionType: 'input', noSeparator: true, defaultAnswer: '' }
+    );
```
Comment (targos) [inline lib/prepare_release.js:276]: Add a link to https://github.com/nodejs/Release/blob/master/CODENAMES.md ?

Comment (richardlau) [reply lib/prepare_release.js]: > 
> 
> Add a link to https://github.com/nodejs/Release/blob/master/CODENAMES.md ?

Done.

**Comment by richardlau** [comment]
14.15.0 (created using this PR in its current state) has now gone out.

**Review by codebytere — APPROVED** [review]

**Review by targos — APPROVED** [review]

**Pull request merged**